frontend/src/data/api.js

```diff
@@ -55,6 +55,17 @@ function doCreateJob(body) {
     }).then(handleResponse);
 }
 
+function doCreateBid(req) {
+    return fetch(properties.APIURLs.createBid, {
+        method: "POST",
+        headers: {
+            ...defaultHeaders,
+            "user-token": window.workhubToken
+        },
+        body: JSON.stringify(req)
```

Comment:
It would be nicer, clearer if we use function parameters rather than passing all req to fetch body.

So, you can take `jobID, bidDescription, bidName` as function arguments and pass those to body like body: `JSON.stringify({ bid_description: bidDescription ... });`